src/Blocks/components/conditional-tags/components/conditional-tags-options.js

```diff
@@ -218,95 +222,90 @@ export const ConditionalTagsOptions = (attributes) => {
+			label={__('Conditional visibility', 'eightshift-forms')}
+		>
+			<>
+			{formFields?.length < 1 ?
+				<IconLabel
+					icon={icons.warningFillTransparent}
+					label={__('Feature unavailable', 'eightshift-forms')}
+					subtitle={__('It looks like your field has a missing name.', 'eightshift-forms')}
+					additionalClasses='es-nested-color-yellow-500!'
+					addSubtitleGap
+					standalone
+				/> :
+				<>
+					<IconToggle
+						label={__('Use conditional visibility', 'eightshift-forms')}
+						checked={conditionalTagsUse}
+						onChange={(value) => {
+							setAttributes({ [conditionalTagsUseKey]: value });
+							setAttributes({ [conditionalTagsRulesKey]: !value ? undefined : [CONDITIONAL_TAGS_ACTIONS.HIDE, []]});
+						}}
+						noBottomSpacing={!conditionalTagsUse}
+						additionalClasses='es-font-weight-500'
 					/>
-				</Control>
-			</PanelBody> :
-			<PanelBody>
-				<IconToggle
-					icon={icons.conditionalVisibility}
-					label={__('Conditional visibility', 'eightshift-forms')}
-					checked={conditionalTagsUse}
-					onChange={(value) => {
-						setAttributes({ [getAttrKey('conditionalTagsUse', attributes, manifest)]: value });
-						setAttributes({ [getAttrKey('conditionalTagsRules', attributes, manifest)]: !value ? undefined : [CONDITIONAL_TAGS_ACTIONS.HIDE, []]});
-					}}
-					noBottomSpacing={!conditionalTagsUse}
-					additionalClasses='es-font-weight-500'
-				/>
 
-				<Section showIf={conditionalTagsUse} noBottomSpacing>
-					{conditionalTagsIsHidden &&
-						<Notification
-							text={__('Your field is hidden! It is important to remember that this may result in unforeseen consequences when used with conditional tags.', 'eightshift-forms')}
-							type={'warning'}
-						/>
-					}
-
-					<Control
-						icon={icons.conditionH}
-						label={__('Rules', 'eightshift-forms')}
-						// Translators: %d refers to the number of active rules
-						subtitle={(rulesCount > 0) && sprintf(__('%d rules', 'eightshift-forms'), rulesCount)}
-						noBottomSpacing
-						inlineLabel
-					>
-						<Button
-							variant='tertiary'
-							onClick={() => setIsModalOpen(true)}
-							className='es-rounded-1.5 es-w-9 es-h-center es-font-weight-500'
-						>
-							{(rulesCount > 0) ? __('Edit', 'eightshift-forms') : __('Add', 'eightshift-forms')}
-						</Button>
-					</Control>
-	
-					{isModalOpen &&
-						<Modal
-							overlayClassName='es-conditional-tags-modal es-geolocation-modal'
-							className='es-modal-max-width-5xl es-rounded-3!'
-							title={<IconLabel icon={icons.conditionalVisibility} label={__('Conditional visibility', 'eightshift-forms')} standalone />}
-							onRequestClose={() => setIsModalOpen(false)}
+					<Section showIf={conditionalTagsUse} noBottomSpacing>
+						{conditionalTagsIsHidden &&
+							<Notification
+								text={__('Your field is hidden! It is important to remember that this may result in unforeseen consequences when used with conditional tags.', 'eightshift-forms')}
```

Comment:
```suggestion
								text={__('Field is hidden. This might introduce issues if used with conditional tags.', 'eightshift-forms')}
```
Why so dramatic? 😂